server/pages/index.js

```diff
@@ -269,6 +271,7 @@ function getDefaultContext( request ) {
 		store: createReduxStore( initialServerState ),
 		bodyClasses,
 		sectionCss,
+		isLoggedIn: !! request.cookies.wordpress_logged_in || isSupportSession( request ),
```

Comment:
will `cookies` always be set here? if not we might want to use `lodash.get()`


```js
isLoggedIn = !! get( request, [ 'cookies', 'wordpress_logged_in' ], isSupportSession( request ) ),
```

Reply:
Other server code seems to assume `request.cookies` is defined, and I believe it will be due to the Express cookie-parser middleware added early in the chain:
https://github.com/Automattic/wp-calypso/blob/5173a231517a0dcf2ad57a36ad15464819932c81/server/pages/index.js#L598